src/components/Modal/Modal.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Modal = (props) => {
+  return props.isOpen ? (
+    <div className="modal">
+      <div className="modal-content--medium">
+        <span onClick={props.handleClose} className="close" />
+        {props.children}
+      </div>
+    </div>
+  ) : (
+    <></>
```

Comment:
Here, in Making Sense, there is a pattern that became very common, that consists of dealing with validation and not happy paths at the beginning and then deal with the more common or useful scenario, something like this:

```js
const Modal = (props) => {
  if (!props.isOpen) {
    return (<></>);
  }

  return (<div className="modal">
      <div className="modal-content--medium">
        <span onClick={props.handleClose} className="close" />
        {props.children}
      </div>
    </div>);
}
```

Reply:
You're right, I forgot to do it